src/ray/core_worker/core_worker.h

```diff
@@ -484,6 +485,23 @@ class CoreWorker {
     }
   }
 
+  /// Returns whether the message was sent to the wrong worker. The right error reply
+  /// is sent automatically.
```

Comment:
could you add a comment on why this can happen, just to make it easier for others to understand the code :)